ui/appui-react/src/appui-react/preview/enable-stratakit-ui/status-bar-message-center/MessageCenterField.scss

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+@use "../../../shared/_MessageCenterCommon";
```

Comment:
Avoid importing anything from current impl.
Custom styling should only use [StrataKit tokens](https://itwin.github.io/design-system/tokens)